repograph/repograph.py

```diff
@@ -50,7 +53,16 @@ def iter_submodules(gh, repo):
             click.echo("Can't find submodule %r" % submodule_name, err=True)
             continue
 
-        yield repo_name, submodule_name
+        sub_version = repo.get_file_contents(path).sha
+
+        sub_head = sub_repo.get_commits()[0].sha
+
+        if sub_head != sub_version:
+            comp = sub_repo.compare(sub_version, sub_head)
+        else:
+            comp = None
```

Comment:
Does `compare` take a long time and/or not work if the commits are equal?  Otherwise it seems simpler to just always return the comparison and handle the equal case in the calling code.